packages/eslint-plugin/lib/config/rules/shopify.js

```diff
@@ -45,6 +45,8 @@ module.exports = {
   '@shopify/react-no-multiple-render-methods': 'off',
   // Prefer all non-React-specific members be marked private in React class components.
   '@shopify/react-prefer-private-members': 'off',
+  // Require input elements to have autocomplete values
+  '@shopify/react-require-autocomplete': 'off',
```

Comment:
Does this make the rule disabled by default, allowing apps to opt in on their own time?

Reply:
Correct.